apps/web/src/state/wallet/hooks.ts

```diff
@@ -311,15 +314,13 @@ export function useCurrencyBalancesWithChain(
 
   const tokens = useMemo(
     () => currencies?.filter((currency): currency is Token => Boolean(currency?.isToken)) ?? [],
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-    [...(currencies ?? [])],
```

Comment:
this might cause issues on development, currently one usage of this hook which is already memoized so it is safe to use currencies directly